### DISCUSSION TITLE
Name Tags with Placeholder Support (and TabList sync)

### DISCUSSION THREAD

**Discussion by hamcoremusic** [Zrips/CMI#1760] [Enhancement]
**Description of issue or feature request:**

It would be amazing if we could edit name tags (above players heads) like the names in the TabList.

For example, I could set: 
```
GeneralFormat:
    PlayerName: '[prefix][playerDisplayName]'
```
just like my tabList.yml and it will sync the name tags and tab list!

Even just to have nick names update would be amazing!

**Comment by mrfloris** [comment]
@Zrips On event servers, and our main survival servers, and servers where we sell ranks, we really wish we could use nicknames, so not just that we can add /cmi nametag for prefix/suffix, so we can see who's ranked how during events and survival stuff, but also if they paid for a nickname, and stuff like that, that their nameplate above their head updates as well.

so "mrfloris" could perhaps also look like [playerDisplayname] so it show as a RED colored mrfloris for admin, or if my nickname is &b1MB&6Floris, that when others stand in front of me they see me with the colored 1mbfloris above my head.

Maybe

`/cmi nameplate -pref:&7king -name:[playerDisplayName] -suf:&7lvl6`

**Comment by hamcoremusic** [comment]
Either command based or even another NamePlate module!

Could make a NamePlate.yml and have

```
##Nameplate Configuration##
Default:
  NameTag: "[playerDisplayName]"
Groups:
  `1`: "[prefix][playerDisplayName]"
```

If that makes any sense!

**Comment by mrfloris** [comment]
Even better, a module, it can be off by default, so others who use a plugin for this don't get bothered, and we can just edit the config for our groups.

**Comment by Yamiru** [comment]
@Zrips hello, I use latest version of CMI and paper 1.21.10 command `/cmi nameplate Yamiru -pref:%luckperms__prefix%` works but config not found for nameplates
